docs/nodes/k8s.md

```diff
@@ -5,50 +5,52 @@ title: K8S
 
 Node classes list of k8s provider.
```

Comment:
I couldn't find the `k8s.ecosystem` in this docs file. I think you should re-run the autogen.sh to update it.

Reply:
Done.